introspection/introspection.go

```diff
@@ -0,0 +1,81 @@
+package introspection
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/wundergraph/graphql-go-tools/pkg/astprinter"
+	wunderintro "github.com/wundergraph/graphql-go-tools/pkg/introspection"
+)
+
+//go:embed introspection.gql
+var introspectionQuery string
+
+type jsonBody struct {
+	Data json.RawMessage `json:"data"`
+}
+
+func Load(url string) ([]byte, error) {
```

Comment:
Could we make the CLI to return a different error code if it can't download the schema? It could be useful to integrate it in the local build process. If not possible or very complicated, I have a plan B

Reply:
@nestor-sk easily doable!

Reply:
Added now!